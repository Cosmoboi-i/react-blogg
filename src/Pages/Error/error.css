.error {
  min-height: 79%;
  display: flex;
  flex-direction: column;
  width: auto;
  justify-content: center;
  align-items: center;
  padding: 1em 0;
}