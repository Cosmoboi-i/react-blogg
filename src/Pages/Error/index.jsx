import React from "react";
import { useParams } from "react-router-dom";
import "./error.css";

export default function Error() {
  const { statusCode } = useParams();

  return (
    <div className="error">
      <h1>{statusCode}</h1>
      <div>Something went wrong!</div>
    </div>
  );
}
