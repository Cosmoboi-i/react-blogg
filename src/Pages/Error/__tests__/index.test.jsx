import { render } from "@testing-library/react";
import React from "react";
import Error from "..";

describe("Error Page", () => {
  it("should render the Error page correctly", () => {
    const { asFragment } = render(<Error />);
    expect(asFragment()).toMatchSnapshot();
  });
});
