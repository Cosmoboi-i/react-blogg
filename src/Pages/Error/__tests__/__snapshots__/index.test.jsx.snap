// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should render the Error page correctly 1`] = `
<DocumentFragment>
  <div
    class="error"
  >
    <h1 />
    <div>
      Something went wrong!
    </div>
  </div>
</DocumentFragment>
`;
